controllers/provider-alicloud/charts/provider-alicloud/templates/configmap-logging.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ include "name" . }}-logging-config
```

Comment:
Check the indentation (on all other configmaps as well), you're using three instead of two spaces